js.js

```diff
@@ -0,0 +1,90 @@
+// Getting references to HTML elements
+const addButton = document.getElementById('addButton');
```

Comment:
You need to use node.js to handle the backend for creating,reading, updating, and deleting of tasks, I think you need to refactor your code, NB: use node.js.

Reply:
Okay will do. Thank you for your review!